CoreFramework/DriverCore/WebDriverAction.cs

```diff
@@ -448,6 +448,32 @@ public void AssertEquals(object actual, object expected)
             throw excep;
         }
     }
+    public void AssertListAscending(List<string> list)
+    {
+        try
+        {
+            list.Should().BeInAscendingOrder();
+            HtmlReport.Pass("List is sorted in ascending order");
+        }
```

Comment:
Is this really needed to write this function?
if you try to use the syntax simply `list.Should().BeInAscendingOrder();`
If it's failed, then after the test is done, it will add the Pass/Fail log for HTML report automatically

Reply:
This is a sample method I wrote for Minh so he can finish his test script for US 306, so I will notify him later. Thank you for your suggestion

Reply:
Done